### PR TITLE
Support kadfloor param in PubMatic adapter

### PR DESCRIPTION
**Pull request by pm-nilesh-chate**
Add support to 'kadfloor' bid param in PubMatic adapter. (Already supported in client side Prebid).

Documentation https://docs.prebid.org/dev-docs/pbs-bidders.html#pubmatic

### CODE REVIEW

**Comment by bsardo** [comment]
Hello. Please open a documentation PR, if you have not already, when you get a chance. https://github.com/prebid/prebid.github.io

**Comment by pm-nilesh-chate** [comment]
> Hello. Please open a documentation PR, if you have not already, when you get a chance. https://github.com/prebid/prebid.github.io

Hi, The documentation is up to date. Refer 'Bid Params' section at https://docs.prebid.org/dev-docs/pbs-bidders.html#pubmatic

File: adapters/pubmatic/pubmatic.go [hunk 1]
```diff
@@ -56,6 +56,7 @@ const (
 	pmZoneIDKeyNameOld = "pmZoneID"
 	ImpExtAdUnitKey    = "dfp_ad_unit_code"
 	AdServerGAM        = "gam"
+	kadfloor           = "kadfloor"
```
Comment (VeronikaSolovei9) [inline adapters/pubmatic/pubmatic.go:59]: Can't see usages of this field

**Review by VeronikaSolovei9 — COMMENTED** [review]
LGTM, just 1 minor comment

File: adapters/pubmatic/pubmatic.go (hunk 1 above)
Comment (pm-nilesh-chate) [reply adapters/pubmatic/pubmatic.go]: removed

**Review by pm-viral-vala — DISMISSED** [review]
LGTM

**Review by VeronikaSolovei9 — APPROVED** [review]
Thank you for addressing to comment, LGTM

File: adapters/pubmatic/pubmatic_test.go [hunk 2]
```diff
@@ -67,3 +71,87 @@ func TestGetBidTypeForUnsupportedCode(t *testing.T) {
 		t.Errorf("Expected Bid Type value was: %v, actual value is: %v", openrtb_ext.BidTypeBanner, actualBidTypeValue)
 	}
 }
+
+func Test_parseImpressionObject(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (pm-nilesh-chate) [reply adapters/pubmatic/pubmatic_test.go]: done

File: adapters/pubmatic/pubmatic_test.go [hunk 3]
```diff
@@ -67,3 +71,87 @@ func TestGetBidTypeForUnsupportedCode(t *testing.T) {
...
+	type args struct {
+		imp                      *openrtb2.Imp
+		extractWrapperExtFromImp bool
+		extractPubIDFromImp      bool
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    *pubmaticWrapperExt
+		want1   string
```
[the remark this answers is not in the document]
Comment (pm-nilesh-chate) [reply adapters/pubmatic/pubmatic_test.go]: done

**Review by VeronikaSolovei9 — APPROVED** [review]

**Review by AlexBVolcy — APPROVED** [review]
LGTM!

**Pull request merged**